src/qt/privacydialog.cpp

```diff
@@ -19,6 +19,7 @@
 #include <QClipboard>
 #include <QSettings>
 #include <utilmoneystr.h>
+#include <qt5/QtWidgets/qwidget.h>
```

Comment:
Not liking this include line, shouldn't `#include <QWidgets>` be enough?

Reply:
Me neither, my IDE does this include stuff without asking :-(
Will fix the issues today after work.